foundations/the_front_end/project_html_css.md

```diff
@@ -80,6 +80,7 @@ Inside your project folder, create your index.html file
    - If you're frustrated with trying to get buttons or inputs to style the way you want (for instance, they seem to just not respond to any styles), look into the CSS property `-webkit-appearance: none;` or `-moz-appearance` if you're using Firefox.
    - [Here's a link to the Google Logo just in case they temporarily replace it with a Doodle!](https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png)
    - [Here's a cached page of the Google home page for reference in case the original logo is not there.](https://web.archive.org/web/20191130234759/https://www.google.com/)
+   - <a href="https://fonts.google.com/icons">Here's a link tot the Google Material Icons page. This is a great resource for free to use open source icons.</a>
```

Comment:
Small typo- "tot the". Also, let's use markdown (see the above bullet), rather than HTML :)